features/service-operation.feature

```diff
@@ -125,6 +125,19 @@ Feature: Command output from service operations like stop/start/restart
     Then the service "docker" should be running
     And have a new pid for "docker" service
 
+    When the "kubernetes" service is running
```

Comment:
I don't think this is going to work as it stands unless Kubernetes is managed by _systemctl_.